#!/usr/bin/env python
PACKAGE = "ping_pong_follower"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("low_H_param"   ,    int_t,    0, "Low H", 8,  0, 255)
gen.add("high_H_param"  ,    int_t,    0, "High H", 105,  0, 255)

gen.add("low_S_param"   ,    int_t,    0, "Low S", 113,  0, 255)
gen.add("high_S_param"  ,    int_t,    0, "High S", 255,  0, 255)

gen.add("low_V_param"   ,    int_t,    0, "Low V", 189,  0, 255)
gen.add("high_V_param"  ,    int_t,    0, "High V", 255,  0, 255)

exit(gen.generate(PACKAGE, "ping_pong_follower", "HSV_thresholding"))
